__tests__/resolvers/exotics/github-resolver.js

```diff
@@ -38,6 +38,17 @@ test('getGitHTTPUrl should return the correct git github SSH url', () => {
   expect(GitHubResolver.getGitSSHUrl(fragment)).toBe(expected);
```

Comment:
Why does the test `getGitHTTPUrl should return the correct git github SSH url` actually run  `GitHubResolver.getGitSSHUrl`?